### DISCUSSION TITLE
Update the Scripts on release CD and other cruft

### DISCUSSION THREAD

**Discussion by morenathan** [gobolinux/Documentation#27] [Ideas]
Trying to install some updates but Scripts tools are pointing as

> Connecting to gobolinux.org (gobolinux.org)|75.119.204.69|:80... connected.
> HTTP request sent, awaiting response... 301 Moved Permanently

Additionally, Firefox is so out-of-date most webpages do not run correctly, including Github.

Sadly there is no LVM MD modules in the included kernel. So I knows there is an LVM partition but not what is on it.

Also, for old people like me with diabetic eyes, such small typefaces are really hard to use.

**Comment by fyrak1s** [comment]
Hi, sorry for the inconvinience, the wiki is down currently and we are working on a solution.
Please follow the [GoboLinux 017 Known Issues and Fixes](https://web.archive.org/web/20240121142222/https://wiki.gobolinux.org/Overview/Installing-Gobolinux/GoboLinux-017-Known-Issues-and-Fixes/) to get yourself up and running.

UPDATE: OUR WIKI IS BACK ONLINE https://wiki.gobolinux.org/

**Comment by rubyFeedback** [comment]
Getting GoboLinux into an installable release would be great. That way testing software or recipes becomes
easier. I track 3810 different programs right now via ruby + YAML files (but I can autogenerate other formats
too, to some extent). Unfortunately the last times I tried to install GoboLinux, I ran into various issues and
gave up eventually.

The "Known Issues and Fixes" also did not cover all problems I encountered. I think it would be better to
consider aiming for an .iso that can be modified (in theory) daily. I use USB discs nowadays and only keep
one DVD drive as an "oldschool" backup. Ideally we could script-automate upgrading the .iso file (and then
the "known issues and fixes" won't be relevant anymore; they are only relevant because nobody has updated
the .iso file so far, right? So that is one bottleneck.)

> for old people like me with diabetic eyes, such small typefaces are really hard to use.

Increasing the font size makes sense in general. Note that failing eyesight is not confined to diabetes; the
ability to focus degrades in general with age, although diabetes naturally may contribute to this situation
more rapidly so.

**Reply by fyrak1s** [reply]
There is nothing inherently stopping you from implementing this, if you need any guidance get in touch with me and I can get you going.

**Reply by morenathan** [reply]
Very true. Never done this and I'm not sure how important it is to me. LFS took that desire years ago.

The scripts package used to be highlighted more prominently on the web site. It was possibly to do this the "Gobo way" inside your own home directory, like Nix can/does now. 

That too seems to have been lost. I did find it all in the GitHub repos, but where to start? Curses, no Wiki!

**Reply by fyrak1s** [reply]
I'd love to guide you but I am unclear about exactly you want to accomplish.

As pointed out previously, the full GoboLinux Wiki is currently available over archive.org: https://web.archive.org/web/20240121085032/https://wiki.gobolinux.org/

UPDATE: OUR WIKI IS BACK ONLINE https://wiki.gobolinux.org/

I am making a note to include LVM modules in the next subrelease.
